content/en/technical-advisory-group/architectural-review-committee/ARC-001.en.md

```diff
@@ -7,4 +7,163 @@ draft: false
+
+#### Key Benefits
+
+✔️ **Unified API surface** for Kubernetes adoption across the GC
+✔️ **Opinionated configuration** that aligns with GC policies and standards
+✔️ **Security-first**, with embedded zero trust and governance by default
+✔️ **Interoperable** with existing departmental infrastructure
+✔️ **Flexible deployment models**, supporting both OSS and proprietary tooling
+✔️ **Strong tenant isolation** via namespace boundaries and enforced network segmentation
+✔️ **Self-service within policy boundaries**, empowering teams while ensuring compliance
+
+### 2.2 Policy-Driven Model
+
+Aurora enables developer autonomy while enforcing non-negotiable guardrails. The platform uses **Gatekeeper (OPA)** and curated policies to ensure all workloads adhere to GC-wide compliance, security, and operational requirements—without the need for manual reviews or central bottlenecks.
+
+By codifying these constraints, Aurora empowers application teams to build, deploy, and manage their solutions independently, while staying within clearly defined, enforceable limits. This **“freedom within boundaries”** model reduces cognitive load on platform and security teams while increasing delivery velocity for application teams.
+
+---
+
+## 3. Governance & Compliance Alignment
+
+The Aurora initiative is supported by the newly established **Technical Advisory Group (TAG)**—a cross-departmental body that provides architectural oversight, technical guidance, and ensures continuous alignment with Government of Canada priorities. Governed by a published **charter, roles, and responsibilities**, the TAG enables community-driven governance and helps shape the platform’s evolution through shared stewardship.
+
+- Aurora **adheres to SSC’s cloud governance model**, ensuring **interoperability, security, and standardization**
+- Leverages **CNCF best practices** for operational maturity
+- Supports **multi-tenancy, workload separation, and automated compliance checks**
+- Aligns with **Treasury Board Secretariat (TBS) cloud directives**
+
+### Key Decision Point for ARC
+
+✔️ Does Aurora meet SSC’s **interoperability and security requirements**?
+✔️ Is the **Technical Advisory Group (TAG)** recognized as a valid mechanism for cross-departmental technical governance?
+
+---
+
+## 4. Operational Model
+
+Aurora’s operational model reflects a scalable, multi-environment architecture grounded in platform engineering principles. It is designed to centralize operational overhead, reduce duplication across departments, and enable secure, self-service cloud-native development at scale. The model supports both shared services and per-department configurations, enabling a flexible yet governed path for onboarding departments of all sizes.
+
+### 4.1 Deployment Approach
+
+Aurora will be deployed in SSC’s **Enterprise Tenant**, with:
+
+- **Two main platform environments** for cluster provisioning and platform services:
+  - `AuroraNonProd` (`mgmt-dev-cc-00`)
```

Comment:
Might be good to mention/elaborate that the use of this environment is to test the cluster provisioning / platform services before they are promoted to the production environment, where all the actual clusters will be managed from.

Just helps get ahead of questions of what the mgmt-dev vs. mgmt-prod manages.

Reply:
Another point too, these bullet points seem somewhat redundant with the points in 4.2. Might be good to try and merge them together to avoid the redundancy.